.github/workflows/client-ios.yml

```diff
@@ -42,8 +42,8 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: true
-      - name: 🔨 Switch to Xcode 14.3
-        run: sudo xcode-select --switch /Applications/Xcode_14.3.app
+      - name: 🔨 Switch to Xcode 15.0
+        run: sudo xcode-select --switch /Applications/Xcode_15.0.app
```

Comment:
by updating xcode you should probably revert #23874, but i wonder that would make test suite ios ci more unstable. would need further figuring out how long it takes to boot up ios 17 simulator.

Reply:
I'll do that and see how long it takes 👍